### PR TITLE
Add script to fix multi comment transcriptions

### PR DESCRIPTION
**Pull request by TimJentzsch**
Relevant issue: N/A

## Description:

Multi comment transcriptions weren't picked up properly by bot, only the first comment made it to Blossom.
This script goes through the transcriptions and fixes them in the database.

:information_source:  This PR is branched from #349, so that should be merged first. :information_source: 

## Checklist:

- [x] Code Quality
- [x] Pep-8
- [ ] Tests (if applicable)
- [x] Success Criteria Met
- [x] Inline Documentation
- [ ] Wiki Documentation (if applicable)

### CODE REVIEW

File: bootstrap/multi-comment-transcriptions.py [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+        push.search_comments(
+            link_id=_get_link_id(transcription),
+            author=author["username"],
+            filter=comment_filter,
+        )
+    )
+    transcriptions: List[CommentData] = [dict_from_comment(x) for x in transcriptions]
+    # Make sure the comments are actually transcriptions
+    transcriptions = [
+        x
```
Comment (itsthejoker) [inline bootstrap/multi-comment-transcriptions.py:82]: tsk tsssskkkk

**Review by itsthejoker — APPROVED** [review]
Looks fine, let's roll!

File: bootstrap/multi-comment-transcriptions.py (hunk 1 above)
Comment (TimJentzsch) [reply bootstrap/multi-comment-transcriptions.py:82]: What's wrong with this? The naming?

Comment (itsthejoker) [reply bootstrap/multi-comment-transcriptions.py:82]: I'm just giving you crap since you've dinged me for single-character variables too 😉

**Pull request merged**